testdata/CheckUntrappableSignal.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func fn() {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	signal.Ignore(os.Kill)            // MATCH /signal cannot be trapped/
+	signal.Notify(c, os.Kill)         // MATCH /signal cannot be trapped/
+	signal.Reset(os.Kill)             // MATCH /signal cannot be trapped/
+	signal.Ignore(syscall.SIGKILL)    // MATCH /signal cannot be trapped/
+	signal.Notify(c, syscall.SIGKILL) // MATCH /signal cannot be trapped/
+	signal.Reset(syscall.SIGKILL)     // MATCH /signal cannot be trapped/
+	signal.Ignore(syscall.SIGSTOP)    // MATCH /signal cannot be trapped/
+	signal.Notify(c, syscall.SIGSTOP) // MATCH /signal cannot be trapped/
+	signal.Reset(syscall.SIGSTOP)     // MATCH /signal cannot be trapped/
+}
```

Comment:
@bradleyfalzon @dominikh Would it make sense to add a test for `os.Signal(syscall.SIGKILL)` and `os.Signal(syscall.SIGSTOP)` ?